bindata/assets/kube-controller-manager/namespace-openshift-infra.yaml

```diff
@@ -4,3 +4,7 @@ metadata:
   annotations:
     workload.openshift.io/allowed: "management"
   name: openshift-infra
+  labels:
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
```

Comment:
The former PR (https://github.com/openshift/cluster-kube-controller-manager-operator/pull/628) only changed the other file and it seemed enough, however, I'm wondering if we should change this file as well.